.github/workflows/ci.yml

```diff
@@ -30,46 +30,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        foreman-core-branch: [1.21-stable, 1.22-stable, 1.23-stable, 1.24-stable, 2.0-stable, 2.1-stable, 2.2-stable, 2.3-stable, develop]
+        foreman-core-branch: [1.24-stable, 2.0-stable, 2.1-stable, 2.2-stable, 2.3-stable, 2.4-stable, develop]
```

Comment:
If we drop support for Foreman < 1.24, do we need to bump the required version in `engine.rb`?

Reply:
true - I missed that.
I think it's fine to do that - we won't be doing any releases for it anyway. WDYT?

Reply:
Sure, let's just bump the version in `engine.rb` and bump the major version in the next release.